cloud/services/compute/instances/reconcile_test.go

```diff
@@ -211,6 +235,21 @@ func TestService_createOrGetInstance(t *testing.T) {
 				Zone: "us-central1-c",
 			},
 		},
+		{
+			name:  "FailureDomain not given (should pick up a failure domain from the cluster)",
+			scope: machineScopeWithoutFailureDomain,
+			mockInstance: &cloud.MockInstances{
+				ProjectRouter: &cloud.SingleProjectRouter{ID: "proj-id"},
+				Objects: map[meta.Key]*cloud.MockInstancesObj{
+					{Name: "my-machine", Zone: "us-central1-c"}: {Obj: &compute.Instance{
+						Name: "my-machine",
+					}},
+				},
+			},
+			want: &compute.Instance{
+				Name: "my-machine",
+			},
```

Comment:
I am not able to understand What I have to specify here in the want :)
Do I have to check a Machine Instance or just check if there is no error something like that